metricbeat/magefile.go

```diff
@@ -51,7 +51,7 @@ import (
 
 func init() {
 	common.RegisterCheckDeps(update.Update)
-
+	devtools.SetBuildVariableSources(devtools.DefaultBeatBuildVariableSources)
```

Comment:
is this needed?
If not, can you remove that line

Reply:
I will remove it..it should not be be there